specification/chaos/resource-manager/Microsoft.Chaos/stable/2023-11-01/experiments.json

```diff
@@ -0,0 +1,547 @@
+          },
+          {
+            "$ref": "./types/common.json#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "./types/common.json#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "./types/experiments.json#/parameters/experimentName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "List of executions returned if request was successful.",
+            "schema": {
+              "$ref": "./types/experiments.json#/definitions/experimentExecutionListResult"
+            }
+          },
+          "default": {
+            "description": "Error response returned if request was unsuccessful.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "List all executions of an Experiment.": {
+            "$ref": "./examples/ListExperimentExecutions.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Chaos/experiments/{experimentName}/executions/{executionId}": {
+      "get": {
+        "description": "Get an execution of an Experiment resource.",
+        "tags": [
+          "Experiments"
+        ],
+        "operationId": "Experiments_GetExecution",
+        "parameters": [
+          {
+            "$ref": "./types/common.json#/parameters/apiVersion"
+          },
+          {
+            "$ref": "./types/common.json#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "./types/common.json#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "./types/experiments.json#/parameters/experimentName"
+          },
+          {
+            "$ref": "./types/experiments.json#/parameters/executionId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Execution returned if request was successful.",
+            "schema": {
+              "$ref": "./types/experiments.json#/definitions/experimentExecution"
+            }
+          },
+          "default": {
+            "description": "Error response returned if request was unsuccessful.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get the execution of a Experiment.": {
+            "$ref": "./examples/GetExperimentExecution.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Chaos/experiments/{experimentName}/executions/{executionId}/details": {
+      "post": {
```

Comment:
If this can be fetched synchronously please change to 

GET
    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Chaos/experiments/{experimentName}/executions/{executionId}/executionDetails/default": {

Reply:
is there particular benefit to use this pattern as opposed to POST? execution detail is one document. We will only ever have one, so it feels like a POST action to fetch execution details is more natural. It'll make more sense to the user

Reply:
The benefit is that the schema of the response will follow the regular ARM envelope semantics and certain feature sets like azure policy will work with ARM resources, they dont work for the responses of post actions